hazenlib/utils.py

```diff
@@ -178,18 +211,16 @@ def get_pixel_size(dcm: pydicom.Dataset) -> (float, float):
 
 
 def get_TR(dcm: pydicom.Dataset) -> float:
-    """
-    Returns Repetition Time (TR)
+    """Get the RepetitionTime field from the DICOM header
 
-    Parameters
-    ----------
-    dcm: pydicom.Dataset
+    Args:
+        dcm (pydicom.Dataset): DICOM image object
 
-    Returns
-    -------
-    TR: float
+    Returns:
+        float: value of the RepetitionTime field from the DICOM header, or defaults to 1000
     """
-
+    # TODO: explore what type of DICOM files do not have RepetitionTime in DICOM header
+    # check with physicists whether 1000 is an appropriate default value
```

Comment:
1000 is fine for a default value. I'm not sure exactly in what scenario this try-except statement would lead to the TR=1000 case, but it's a 'sensible' default nonetheless.

You can remove this TODO